pkg/controller/schedulingpreference/controller.go

```diff
@@ -76,8 +72,8 @@ type SchedulingPreferenceController struct {
 // SchedulingPreferenceController starts a new controller for given type of SchedulingPreferences
 func StartSchedulingPreferenceController(config *util.ControllerConfig, schedulingType schedulingtypes.SchedulingType) (schedulingtypes.Scheduler, error) {
 	userAgent := fmt.Sprintf("%s-controller", schedulingType.Kind)
-	fedClient, kubeClient, crClient := config.AllClients(userAgent)
-	controller, err := newSchedulingPreferenceController(config, schedulingType.SchedulerFactory, fedClient, kubeClient, crClient)
+	_, kubeClient, _ := config.AllClients(userAgent)
```

Comment:
(No action required) Is this a hint that it should be possible to initialize just the generic client from the controller config?